libexec/setters/set-ilks-chop

```diff
@@ -42,6 +42,7 @@ for token in $tokens; do
             setChop "$(jq -r ".tokens.${token}.ilks.${ilk} | .flipDeploy.chop" "$CONFIG_FILE")" "$MCD_CAT"
         else
             setChop "$(jq -r ".tokens.${token}.ilks.${ilk} | .clipDeploy.chop" "$CONFIG_FILE")" "$MCD_DOG"
+            sethSend "$(eval echo "\$MCD_CLIP_${token}_${ilk}")" 'upchost()'
```

Comment:
So, now there's a subtle ordering requirement...`set-ilks-chop` needs to be executed after `set-ilks-dust` for `chost` to end up correct. Maybe we should at least add comments somewhere to document this? I wouldn't be opposed to just calling `upchost` in both places, except for gas costs.

Reply:
Added a comment in base-deploy file about this ordering